content/blog/reliable-local-development-for-infrequently-used-projects.md

```diff
@@ -0,0 +1,143 @@
+---
+id: blog
+title: Reliable codified development setups with GitPod
+description: Local development setups are hard, and they often don't age well. This blog
+  outlines how you can fix that with codified cloud-based development setups using GitPod
+author: Benny Cornelissen
+tags:
```

Comment:
Also, in terms of SEO and findability (both of which I'm an absolute novice in), would it be valuable to add GitPod as a tag here too? Or is having GitPod in the title enough?

Reply:
We don't currently use many tech-specific tags in our website. We may consider doing that but I'm about as novice in SEO as you are probably. Maybe we should park that for now, learn about SEO, and then update all blogs with proper tagging later on?

Reply:
Agreed!